src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/controllers/legaladvisor/GenerateOrderCallbackHandlerTest.java

```diff
@@ -51,7 +51,7 @@
         "doc_assembly.url=http://doc-assembly-api"
     }
 )
-public class GenerateOrderCallbackHandlerTest extends BaseMockSpringTest {
+public class    GenerateOrderCallbackHandlerTest extends BaseMockSpringTest {
```

Comment:
Minor but there is extra whitespace here. This will probably get picked up by the checkstyle.

Reply:
actioned